docs/sop/gather-cluster-logs.md

```diff
@@ -0,0 +1,136 @@
+# Collect logs from Online clusters
+
+## Synopsis
+
+````
+  ssh -i online/logs_access_key opsmedic@use-tower2.ops.rhcloud.com \
+      -- -u USER -c CLUSTER [-n node1 node2...] > logs.tar.gz
+````
+
+**NOTE**: the collected logs are dumped to the standard output as a compressed
+tarball, so **you must redirect the output of ssh** to a file of your choice.
+
+## SSH key to access logs
+
+Access to logs is done via ssh to a bastion host as the *opsmedic* user with a
+specific SSH key that is used for this purpose only. This SSH key is recreated
+on a weekly basis and stored in the *rotating_keys* branch of the
+[shared-secrets repo](https://github.com/openshift/shared-secrets/tree/rotating_keys).
+
+## Information collected
+
+The [log gathering script](../../tower-scripts/bin/gather-logs.sh) collects:
+
+* Master info:
+
+   - journal for these services: `atomic-openshift-master-{api,controllers}` and `etcd`
+   - The output of `oc get node`
+   - The output of the `/metrics` endpoint
+
+* Selected nodes info:
+
+   - journal for these services: docker, atomic-openshift-node, dnsmasq, openvswitch, ovs-vswitchd, ovsdb-server
+   - The output of `oc describe node` for that node
+   - The output of the `/metrics` endpoint for that node (via API proxy)
+
+## Command usage
+
+The log gathering command accepts 3 parameters:
+
+- `-u username` (**mandatory**): specify your kerberos id here.
```

Comment:
Just a comment on terminology.  "mandatory", or "required"?

They're synonymous in this context but I don't recall "mandatory" being used in man pages, etc for this meaning.